cpp/src/layout/legacy/fa2_kernels.cuh

```diff
@@ -83,16 +106,18 @@ void apply_attraction(const vertex_t* restrict row,
                       bool lin_log_mode,
                       const float edge_weight_influence,
                       const float coef,
+                      bool prevent_overlapping,
+                      const float* restrict vertex_radius,
                       cudaStream_t stream)
 {
   // 0 edge graph.
   if (!e) return;
 
   dim3 nthreads, nblocks;
-  nthreads.x = min(e, CUDA_MAX_KERNEL_THREADS);
+  nthreads.x = min(static_cast<int>(e), CUDA_MAX_KERNEL_THREADS);
```

Comment:
Would this be better as:
`nthreads.x = static_cast<int>(e, CUDA_MAX_KERNEL_THREADS)`?

If e doesn't fit in in a 32-bit integer we'll want to be sure we pick CUDA_MAX_KERNEL_THREADS.

If the above doesn't work, something like this might work:

`nthreads.x = static_cast<int>(e, static_cast<edge_t>(CUDA_MAX_KERNEL_THREADS))`?